package.json

```
@@ -0,0 +1,85 @@
+{
+  "author": "JakartaGameStudio",
+  "name": "reed.green.light",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "npm run build:prod && npm run server",
+    "dev": "webpack serve --open",
+    "server": "node ./server/index.js",
+    "build": "webpack",
+    "build:prod": "webpack --mode=production",
+    "lint:all": "npm-run-all lint:scripts",
+    "lint:scripts": "eslint ./src ./server ./typings ./config --fix"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.0",
```

❗️
Версии нужно запинить
```suggestion
    "@babel/core": "7.16.0",
```

Чтобы больше не натыкаться на это, можно запилить `.npmrc` файлик, где явно указать опцию `save-exact=true`. Почему надо пинить? Потому что код, на который опираются наши проекты, пишут люди, которые могут ошибаться, и нас это затрагивать не должно. В качестве общего правила можно думать об этом так: все манипуляции с версиями используемых зависимостей должны быть намеренными и точными